presto-docs/src/main/sphinx/sql/create-function.rst

```diff
@@ -25,11 +25,18 @@ Description
 
 Create a new function with the specified definition.
 
-Each function is uniquely identified by its qualified function name
+When ``TEMPORARY`` is specified, the created function is valid and visible
+within the current session, but no persistent entry is made.
```

Comment:
Do we have a link to what "current session" is? Also, is there a TTL?

Reply:
"Session" mostly makes sense in the context of the CLI. There is no TTL, the scope of the function is the lifespan of the session.